Update documentation for cleaning up Runtime data (#6503)
kyma-project/kyma: docs/compass/03-01-provisioner-details.md

    
Note that the operations of provisioning and deprovisioning are asynchronous. They return the operation ID, which you can use to [check the Runtime Operation Status](#tutorials-check-runtime-operation-status).

The Runtime Provisioner also allows you to [clean up Runtime data](#tutorials-clean-up-runtime-data). This operation removes the data for a given Runtime from the database and frees up the Runtime ID for reuse. It is useful when your cluster has died or when the operation of deprovisioning has failed.

> > **CAUTION:** The operation of cleaning up Runtime data does not trigger Runtime deprovisioning and the cluster might still exist after running it.

Reviewer: ```suggestion
> **CAUTION:** Cleaning up Runtime data does not trigger Runtime deprovisioning and the cluster might still exist after the cleanup.
```
